docs/options.md

```diff
@@ -69,6 +69,14 @@ See the [strings rationale](rationale.md#strings) for more information.
 | ------- | ---------------- | --------------------- |
 | `false` | `--single-quote` | `singleQuote: <bool>` |
 
+## JSX Quotes
+
+Use single quotes in JSX
+
+| Default | CLI Override | API Override   |
+| ------- | ------------ | -------------- |
+| `false`  | `--jsx-single-quote`  | `jsxSingleQuote: <bool>` |
```

Comment:
I think you need to run Prettier on this file to make `yarn lint-docs` pass :)

Reply:
I had no idea that was a thing, it's not part of the [Contributing docs](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md) :trollface: . Happy `yarn lint-docs --fix` works like a charm 👍